addons/l10n_fi_rapko/data/account_account_template_data.xml

```diff
@@ -0,0 +1,5394 @@
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">Lis&#228;poistot</field>
+    </record>
+    <record model="account.account.template" id="a116106">
+        <field name="code">116106</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">Poistoeron muutos</field>
+    </record>
+    <record model="account.account.template" id="a116107">
+        <field name="code">116107</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">Poistoero</field>
+    </record>
+    <record model="account.account.template" id="a116200">
+        <field name="code">116200</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">Televiestint&#228;- ,audio- ,video, muut ICT</field>
+    </record>
+    <record model="account.account.template" id="a116201">
+        <field name="code">116201</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">Lis&#228;ykset</field>
+    </record>
+    <record model="account.account.template" id="a116202">
+        <field name="code">116202</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">V&#228;hennykset</field>
+    </record>
+    <record model="account.account.template" id="a116203">
+        <field name="code">116203</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">Katettu j&#228;lleenhankintavarauksella</field>
+    </record>
+    <record model="account.account.template" id="a116204">
+        <field name="code">116204</field>
+        <field name="user_type_id" ref="account.data_account_type_current_assets"/>
+        <field name="chart_template_id" ref="chart_template_fi_rapko"/>
+        <field name="reconcile" eval="False"/>
+        <field name="name">Poistot</field>
+    </record>
+    <record model="ac
```

Comment:
@mlaitinen 

can you suggest correction for below mentioned accounts ?

see line number 5085, it uses same account code/xmlID but with different name than this one.
same thing with the accounts defined on line numbers 5057 and 5092.

Reply:
This is a closed PR. We're working on the improvements for this module. It won't target `10.0` but `12.0`.

I'd suggest you to forget about this PR.

Reply:
@mlaitinen 
do you have a branch with those improvements ? If not, can you please create one and provide us the same ? Actually we are looking to add Finland localization in official version, so, if you can provide us branch it would be easier for us to review and process it.
Thanks.

Reply:
Here it is: https://github.com/avoinsystems/l10n_fi

Reply:
And now here as well #31016 @hsh-odoo